datasketches-memory-java8/src/main/java/org/apache/datasketches/memory/Memory.java

```diff
@@ -35,67 +33,76 @@
 /**
  * Defines the read-only API for offset access to a resource.
  *
- * <p>The classes in this package are not thread-safe.</p>
```

Comment:
Rather than having this in every class, this comment applies to the whole package and this comment is already in the package-info file, which makes this statement redundant.